student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly make sure to not repeat the values of the parent constructor parameters by default `name: name,` is the same as `name,`

```suggestion
    super(age, name, parent_permission)
```

Reply:
cannot use this suggestion because it gives a wrong number of argument error.rather than this I had to use a array which contains default values.
![image](https://github.com/yashodhicy/SchoolLibrary/assets/116276432/b42d8f78-27d2-487d-b820-f131edbb7e5a)

Reply:
@yashodhicy, did you try to use `name = 'Unknown', parent_permission = true` in the initialize method instead